<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GlobalSettings</class>
 <widget class="QDialog" name="GlobalSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Global settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Use &amp;TCP ports</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>&amp;From:</string>
              </property>
              <property name="buddy" >
               <cstring>tcpFrom</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tcpFrom" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum" >
               <number>65535</number>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>5</vsizetype>
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>&amp;to:</string>
              </property>
              <property name="buddy" >
               <cstring>tcpTo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="tcpTo" >
              <property name="sizePolicy" >
               <sizepolicy>
                <hsizetype>1</hsizetype>
                <vsizetype>0</vsizetype>
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximum" >
               <number>65535</number>
              </property>
              <property name="minimum" >
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>&amp;Public network address</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="paAutodetect" >
            <property name="text" >
             <string>&amp;Autodetect</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QRadioButton" name="paManual" >
              <property name="text" >
               <string>&amp;Manual:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="publicAddress" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>&amp;FTP</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>&amp;Advanced</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="ftpUseUnicode" >
            <property name="text" >
             <string>Use &amp;unicode by default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ftpAllowPassive" >
            <property name="text" >
             <string>Allow &amp;passive connections</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>tcpFrom</tabstop>
  <tabstop>tcpTo</tabstop>
  <tabstop>paAutodetect</tabstop>
  <tabstop>paManual</tabstop>
  <tabstop>publicAddress</tabstop>
  <tabstop>ftpUseUnicode</tabstop>
  <tabstop>ftpAllowPassive</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>GlobalSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>409</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GlobalSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>500</x>
     <y>313</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
